README.md

```diff
@@ -139,10 +139,8 @@ In addition to code blocks, some code blocks should be rendered differently depe
 ### Example
```

Comment:
This suggestion means to add some explanation for the **Example**.
```suggestion
### Example
When you write a md code as follows, it looks like **How it looks** when rendered:
```

Reply:
Satendra-SR:Fixed-readme-formatting